docs/markdown/renderer.tsx

```diff
@@ -105,8 +105,7 @@ const Heading = (props: HeadingProps) => {
     store.add(nodeKey, { key: nodeKey, label, level, path: `#${slug}` });
   }
   const css = {
-    marginTop: 0,
-    '&:not(:first-of-type)': { marginTop: 30 },
+    '&:first-child': { marginTop: 0 },
```

Comment:
What is the reason behind changing this (other than simplifying things?)

Reply:
the previous style would result in:
``` tsx
// 0
<h1/>

// 0
<h2/>

// 0
<h3/>

// 30
<h2/>

// 30
<h3/>
```

but i think the intention was to just do:
``` jsx
// 0
<h1/>

<h2/>

<h3/>

<h2/>

<h3/>
```